### PR TITLE
transfer lcsh over

### PR DESCRIPTION
**Pull request by fenekku**
- add LCSH generation for InvenioRDM
- release: v0.2.0

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changes
 
+Version 0.1.0 (released 2023-11-06)
```
Comment (sharpattack) [inline CHANGES.md:3]: Did you mean `0.2.0`?

**Review by sharpattack — APPROVED** [review]
Looks good to me, thanks for adding tests.

File: CHANGES.md (hunk 1 above)
Comment (fenekku) [reply CHANGES.md:3]: Ah yes, thank you!

**Review by mattbaumann1 — APPROVED** [review]
Looks good to me.

**Review by mattbaumann1 — COMMENTED** [review]
Looks good

**Pull request merged**